handbook/direction/code-graph/index.md

```diff
@@ -0,0 +1,56 @@
+# Code Graph
+
+The Code Graph org’s builds and scales the Code Graph and how our users interact with it. We are responsible for search (the user experience as well as performance), insights, code intelligence, and batch changes. This page describes overarching initiatives that tie multiple teams and milestones within Code Graph together.
+
+Individual team direction pages within Code Graph can be found on our [direction home page](../index.md#code-graph). For details on how the themes here become reality, as well as what each team is working on next for their individual visions, take a look there.
+
+## Mission
+
+What is the team's reason for existing? What is the fundamental value they aim to provide? Who are your key customers/stakeholders and why, and are there any customers/stakeholders that you're explicitly choosing not to address that people might have expected you to?
```

Comment:
I think the mission of the code graph org, as owners of what is arguably the core product, is the mission of the company: https://about.sourcegraph.com/handbook/company/strategy#mission. We want to provide a search and code interaction experience that enables everyone to code. The cloud team provides the growth engine that generates stability for the company, enablement supports us all, but we provide the product features that make the company mission reality.

What does everyone think?

Reply:
In general, I think we're the foundation of the `build great, valuable features -> get more users -> convert them to paid` funnel.

Reply:
> We want to provide a search and code interaction experience that enables everyone to code.

I agree with that. Other thoughts I had in that direction that are more specific than "enable everyone to code" (and I'm not sure whether specificity is good or bad here):

- We want to build a product that makes working with code better, independent of the size and rate of growth of the codebase
- Our goal is to enable everyone in every codebase to understand & change the code they're working with

Nothing here rolls off the tongue, I'll admit that, but I think having something separate from the mission of the company is valuable.

Reply:
Yeah agree, I like where these are headed.

Reply:
Maybe something in addition to the above two like the following:

- The Code Graph org provides the main features which people use every day to get value out of the product, so it's important for us to focus on delivering paid features that customers feel are worth paying for.
- Free features are also important, because they give people the chance to try Sourcegraph, and see how valuable it is. But, the ultimate purpose of this part of the funnel is eventually converting them into paid users.

These still need wordsmithing too, but the sentiment is there.

I think it would also be nice to also tie the second bullet from your ideas @mrnugget to the company mission.

Reply:
Applied changes to text